### ISSUE TITLE
https/SSL secure connection

### ISSUE THREAD

**Issue by shaief** [publiclab/plots2#58]
....

**Comment by jywarren** [comment]
OK, tackling this now. So, the i.publiclab.org stuff was the main reason we were waiting besides the chat links. The globalsign folks haven't gotten back to me on it, and so I'll get in touch again, but if they don't have a wildcard cert, I wonder if we could just apply for a second cert for i.publiclab.org?

The alternative, unfortunately, might be to just change all the i.publiclab.org links, but sadly they're peppered through the actual site content (this would be a *good* thing if we used a CDN). Thoughts, Bryan?

**Comment by jywarren** [comment]
So, as a test, I turned on forced SSL, but it causes errors to show in the browser bar in Chrome, and Firefox shows "This is an untrusted ..." and actually forces you to click through a couple steps before displaying the site at all. Internet Explorer 8 says not much, it seems:

![image](https://cloud.githubusercontent.com/assets/24359/5856749/f33886ac-a213-11e4-8fd5-a9e9538ed270.png)

Still, w/ the Firefox non-display, it seems we have to fix i.publiclab.org before moving forward. Also, unfortunately, Chrome now "knows" there's an SSL option and tries to default to it, which of course doesn't work anymore since I stopped running it. It took a cache clear to get http://publiclab.org to start loading again without trying to redirect to https.
